build/linux/installer/scripts/livenessprobe.sh

```diff
@@ -129,7 +129,9 @@ else
       # echo "Telegraf is not running" > /dev/termination-log
       echo "Telegraf is not running (controller: ${CONTROLLER_TYPE}, container type: ${CONTAINER_TYPE})" > /dev/write-to-traces  # this file is tailed and sent to traces
       ## Remember to evaluate all the wait times and polls we have in main.sh before enabling telegraf for livenessprobe
-      # exit 1
+      if [ "${AZMON_TELEGRAF_LIVENESSPROBE_ENABLED}" == "true" ] && [ "${SIDECAR_SCRAPING_ENABLED}" == "true" ]; then
```

Comment:
This should be 

 if [ "${AZMON_TELEGRAF_LIVENESSPROBE_ENABLED}" == "true" ] && [ "${MUTE_PROM_SIDECAR}" != "true" ]; then